app/src/main/java/com/codingblocks/cbonlineapp/activities/VideoPlayerActivity.kt

```diff
@@ -236,17 +233,19 @@ class VideoPlayerActivity : AppCompatActivity(),
         super.attachBaseContext(ViewPumpContextWrapper.wrap(newBase))
     }
 
-    override fun onItemClick(position: Int, id: String) {
-        if (contentId == id) {
-            if (displayYoutubeVideo.view?.visibility == View.VISIBLE)
-                youtubePlayer?.seekToMillis(position * 1000)
-            else
-                videoPlayer?.seekTo(position.toLong() * 1000)
+    private fun handleNoteClick() {
+        viewModel.selectedNote.observer(this) {
```

Comment:
no need of this

Reply:
@aggarwalpulkit596 That is how the architecture works. Please Check this example, https://github.com/alphamu/android-MVVM-DataBinding-RecyclerViewExample. This will clear things out